### DISCUSSION TITLE
Fresh remix repo has hydration issues on Digital Ocean when links are used - how can I fix this?

### DISCUSSION THREAD

**Discussion by MoSattler** [remix-run/remix#5426] [Q&A]
Reproduction steps:

### Prepare repo
1. `npx create-remix@latest repro`
1. run `yarn` 
1. add some links, [like here](https://github.com/teamlearns/remix-repro-repo/blob/main/app/root.tsx#L12-L14)

I did everything here, so you can just clone that: https://github.com/teamlearns/remix-repro-repo

### Deploy
1. Deploy to [Digital Ocean Apps](https://cloud.digitalocean.com/apps)
1. Go to deployed url and check console. 

It will show hydration errors. 

If I remove the link, the hydration issue disappears. 

Anyone has any idea how I can get this to work?

**Comment by clgeoio** [comment]
Do you have any browser extension enabled?
If so, this is a symptom of an existing issue that I've created a proposal for [here](https://github.com/remix-run/remix/discussions/5244)

**Reply by MoSattler** [reply]
No, I don't. And in fact it works fine if I don't use any links like [this](https://github.com/teamlearns/remix-repro-repo/blob/main/app/root.tsx#L12-L14). It also works fine when being served locally, so I doubt it's the browser.

But could it be possible that Digital Ocean is injecting something in the markup on load?
